_posts/2018-11-15-using-vault-with-strimzi.md

```diff
@@ -0,0 +1,198 @@
+---
+layout: post
+title:  "Using Vault PKI secrets engine as a CA with the Strimzi Kafka Operator"
+date: 2018-11-15
+author: chris_vest
+---
+
+<!--more-->
```

Comment:
try to put an introduction here before the "more" separator. It will be shown in the blog posts list so that people don't see only the title. They have a hint on what is the post about then can click to read the entire post.